packages/pwa/app/components/_app/index.jsx

```diff
@@ -168,6 +176,25 @@ const App = (props) => {
                             href={getAssetUrl('static/img/global/apple-touch-icon.png')}
                         />
                         <link rel="manifest" href={getAssetUrl('static/manifest.json')} />
+
+                        {/* Urls for all localized versions of this page (including current page)
+                        For more details on hrefLang, see https://developers.google.com/search/docs/advanced/crawling/localized-versions */}
+                        {SUPPORTED_LOCALES.map((locale) => (
+                            <link
+                                rel="alternate"
+                                hrefLang={locale.toLowerCase()}
+                                href={`${appOrigin}${buildUrlLocale(locale)}`}
+                                key={locale}
+                            />
+                        ))}
+                        {/* A general locale as fallback. For example: "en" if default locale is "en-GB" */}
+                        <link
+                            rel="alternate"
+                            hrefLang={defaultLocale.slice(0, 2)}
+                            href={`${appOrigin}${buildUrlLocale(defaultLocale)}`}
```

Comment:
Why do we need this fallback as we already have the `x-default` below? Is it possible that we can have duplicated link if the `defaultLocale` also exists in the `SUPPORTED_LOCALES`? 🤔

Reply:
[Google](https://developers.google.com/search/docs/advanced/crawling/localized-versions#all-method-guidelines) suggests having 2 fallbacks. You can think of them as one smaller and another being the wider fallback.

- one general locale, which is for example `en` (not the specific `en-US`, `en-GB` etc)
- and x-default locale to catch any other unsupported user locales.

Reply:
And speaking of duplicates, we need to look at the `hrefLang` values instead. Looking at it that way, there is actually no duplicates.